app/templates/partials/messages.html

```diff
@@ -4,13 +4,13 @@
 {% from './components/lists/_macro.njk' import onsList %}
 
 {% if messages | length == 1 %}
-    {% if page_url == '/start/' %}
```

Comment:
Previously wrong with only Start page having a difference but now Start journey is different to others? Seems strange having inconsistency on different journeys.